acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeFactory.java

```diff
@@ -161,6 +168,39 @@ public BesuNode createNodeWithP2pDisabled(final String name) throws IOException
             .build());
   }
 
+  public BesuNode createNodeWithMultiTenancy(
+      final String name,
+      final String enclaveUrl,
+      final String authFile,
+      final String authPubKeyFile,
+      final String authPrivateKeyPath)
+      throws IOException, URISyntaxException {
+    final PrivacyParameters.Builder privacyParametersBuilder = new PrivacyParameters.Builder();
+    final PrivacyParameters privacyParameters =
+        privacyParametersBuilder
+            .setMultiTenancyEnabled(true)
+            .setEnabled(true)
+            .setStorageProvider(new InMemoryPrivacyStorageProvider())
+            .setEnclaveFactory(new EnclaveFactory(Vertx.vertx()))
+            .setEnclaveUrl(URI.create(enclaveUrl))
+            .setEnclavePublicKeyUsingFile(
```

Comment:
is this needed for multi-tenancy (i.e. isn't this in the auth file?)

Reply:
The same key is present in both files, but they're in different formats for different consumers.